src/reanimated2/animation/util.ts

```diff
@@ -43,6 +44,29 @@ if (__DEV__ && IS_REDUCED_MOTION) {
   );
 }
 
+export function assertEasingIsWorklet(
+  easing: EasingFunction | EasingFunctionFactory
+) {
+  'worklet';
+
+  if (isWeb()) {
```

Comment:
This is the pattern we currently use throughout the code.
```suggestion
const SHOULD_BE_USE_WEB = SHOULD_BE_USE_WEB();

export function assertEasingIsWorklet(
  easing: EasingFunction | EasingFunctionFactory
) {
  'worklet';

  if (SHOULD_BE_USE_WEB) {
```